pages/Identity/Registry.md

```diff
@@ -53,6 +57,12 @@ So punycode does not properly present non-Latin characters which isn't reaching
 
 Clients resolving handles MUST implement a method to detect potential homographs and check both user settings and potentially check the registry for additional potential matching DSNP Ids.
 
+### EIP 712 Methods, Replay Attacks and Nonces
+
+The Registry supports EIP 712 methods to permit a second party to pay gas costs for address and handle changes. Once an EIP 712 transaction is made, anyone may replay that action without further authorization. This breaks our security guarantees when the registration owner has made an additional change to either address or handle. To mitigate this, the Registry contract MUST store a nonce for every registration. When it receives an EIP 712 transaction, it MUST check that the nonce parameter matches the stored nonce, and it MUST increment the stored nonce if the transaction succeeds.
```

Comment:
Could you break out each sentence on it's own line? As long as it's only a single newline, it won't show up on the rendered page, but it makes diffs easier to read.